Frizz/ViewController.swift

```diff
@@ -9,11 +9,39 @@
 import UIKit
 
 class ViewController: UIViewController {
-
+    
+    @IBOutlet weak var frizz: UILabel!
+    @IBOutlet weak var gourmetFries: UILabel!
+    @IBOutlet weak var backgroundImage: UIImageView!
+    
+    
     override func viewDidLoad() {
         super.viewDidLoad()
-        // Do any additional setup after loading the view, typically from a nib.
+        
+        //Adding blur effect
+        let blurEffect = UIBlurEffect(style: .regular)
+        let blurEffectView = UIVisualEffectView(effect: blurEffect)
+        
+        //Always fill the view
+        blurEffectView.frame = backgroundImage.bounds
+        blurEffectView.contentMode = .scaleToFill
+        backgroundImage.addSubview(blurEffectView)
+    }
+    
+    
+    @IBAction func toLogin(_ sender: UIButton) {
```

Comment:
What does this function do ?

Reply:
you press the button login and it checks the conditions first before sending you the next stage